docs/USERS-GUIDE.md

```diff
@@ -1,40 +1,41 @@
-# DAPLink Users Guide
-There are three interfaces that DAPLink provides. These are drag-n-drop programming, a serial port and debugging support. In addition DAPLink firmware can be updated using the drag-n-drop programming interface of the bootloader.
+# DAPLink users guide
 
```

Comment:
@brianesquilona Could you please add an introduction about what DAPLink is and why you'd use it?

Reply:
@AnotherButler , can I redirect you to the readme file :) [The very first paragraph] (https://github.com/ARMmbed/DAPLink/blob/master/README.md)

Reply:
Aha. I was looking at the README in docs, not in the overall repo.